LocalWiki.php

```diff
@@ -419,5 +419,9 @@ function onBeforePageDisplay( OutputPage $out ) {
 	case 'worldboxwiki':
 		$wgSpecialPages['Analytics'] = DisabledSpecialPage::getCallback( 'Analytics', 'MatomoAnalytics-disabled' );
 
+		break;
+	case 'meowpediawiki':
+		$wgCleanSignatures = false;
```

Comment:
This should go on LocalSettings.php instead imo. You can also just add it to ManageWiki if you want, but that's up to you.

Reply:
Wouldn’t adding to LS.php set false as default for all wikis?

Reply:
You can set the default value to true and then set the set it to false for that wiki only but in this case, it makes more sense to just add it into ManageWiki

Reply:
As long as the default remains true, I have no objections. ManageWiki can be done.

Reply:
Just noting that even if you're going the ManageWiki route, the default still needs to be set at LocalSettings, else it doesn't work properly.